### ISSUE TITLE
Macro in C++ causing bad syntax highlighting

### ISSUE THREAD

**Issue by jasjuang** [sublimehq/Packages#530]
Steps to reproduce

Create an empty test.h file and paste the below code in it

``````
#include <iostream>
MYMACRO

namespace wa
{
class test
{
 public:
  func();
};
}
``````
Comparing with
``````
#include <iostream>

namespace wa
{
class test
{
 public:
  func();
};
}
``````
the difference is obvious.

I am on Ubuntu 14.04 LTS.

**Comment by wbond** [comment]
This should be fixed by b25a8ef0590d6a57c364aa8e1a2f8fa6e1af315c

**Comment by jasjuang** [comment]
Thanks so much for the fix. Look forward to the next release!
